app/models/dependent.rb

```diff
@@ -101,8 +148,49 @@ def age_at_end_of_year(tax_year)
   end
 
   def eligible_for_child_tax_credit?(tax_year)
-    # add additional eligibility logic
-    age_at_end_of_year(tax_year) < 17
+    is_qualifying_child? && age_at_end_of_year(tax_year) < 17
```

Comment:
haha it seems optimistic and probably unnecessary to pass in tax year given that we only know the policy for one tax year. if in future we have other tax years to compute the CTC for, the rules might be different. not a big deal code-wise, but it might be simpler to remove the argument

Reply:
oh and now that i've read further, `is_qualifying_child?` is already hard-coding 2020, so it probably won't make sense to use that method in conjunction with an age cutoff that applies to any other year